libs/orchestration/terraform-azurerm-amido-monitoring/vars.tf

```diff
@@ -0,0 +1,45 @@
+variable "location" {
+  default = "UK South"
+}
+
+variable "alert_name" {
+}
+
+variable "resource_group_name" {
+}
```

Comment:
see comment above about generating docs
if you look at the other modules they all have descriptions - these are then used by the doc generator

Reply:
done, I think